src/product/billOfLading/billOfLadingV1CarrierItem.ts

```diff
@@ -0,0 +1,138 @@
+import { cleanSpecialChars, floatToString } from "../../parsing/common";
+import { StringDict } from "../../parsing/common";
+import { Polygon } from "../../geometry";
+
+/**
+ * The goods being shipped.
+ */
+export class BillOfLadingV1CarrierItem {
```

Comment:
Same remark as for non-list objects. Could be interesting to inherit from an abstract base class.

Reply:
Noted :+1: